docs/Branches.md

```diff
@@ -18,11 +18,11 @@ To switch from one set of branches to another, you can use `utils/update-checkou
 
 ## The Release Branches
 
-| Swift            | LLVM Project
-| ---------------- | ----------------------
-| swift-x.y-branch | swift/swift-x.y-branch
+| Swift       | LLVM Project
+| ----------- | -----------------
+| release-x.y | swift/release-x.y
 
-At some point before a release, a *release branch* will be created in every repository with a name like `swift-4.0-branch`. (The actual number is chosen by Apple.) After the branch has been created, commits must make it to this branch to make it into the release. In some cases, the [release manager][] for the branch will decide to merge in all additional changes from `master`; otherwise, cherry-picking changes and making a new pull request is the way to go. If there are any "patch" releases (e.g. Swift 4.0.1), they will also come from this branch.
+At some point before a release, a *release branch* will be created in every repository with a name like `release-5.3`. (The actual number is chosen by Apple.) After the branch has been created, commits must make it to this branch to make it into the release. In some cases, the [release manager][] for the branch will decide to merge in all additional changes from `master`; otherwise, cherry-picking changes and making a new pull request is the way to go. If there are any "patch" releases (e.g. Swift 5.3.1), they will also come from this branch.
```

Comment:
```suggestion
At some point before a release, a *release branch* will be created in every repository with a name like `release/5.3`. (The actual number is chosen by Apple.) After the branch has been created, commits must make it to this branch to make it into the release. In some cases, the [release manager][] for the branch will decide to merge in all additional changes from `master`; otherwise, cherry-picking changes and making a new pull request is the way to go. If there are any "patch" releases (e.g. Swift 5.3.1), they will also come from this branch.
```

it should be `release/x.y`